src/ONYXKEYS.js

```diff
@@ -53,8 +50,12 @@ export default {
     BETAS: 'betas',
 
     // NVP keys
+    // Contains the user's payPalMe address
     NVP_PAYPAL_ME_ADDRESS: 'nvp_paypalMeAddress',
 
+    // Contains the user preference for the LHN priority mode
+    NVP_PRIORITY_MODE: 'nvp_priorityMode',
```

Comment:
We might want to add a migration for this. Should be pretty simple copy/paste of the "rename" that we did here

https://github.com/Expensify/Expensify.cash/blob/master/src/libs/migrations/RenameActiveClientsKey.js

Reply:
But we could also skip it since we are still in beta and the worst case scenario is that people will just have to reset their priority mode settings.

Reply:
yeah I wasn't sure if it was really significant enough to require a change since it wouldn't break the UI like the `timezone` change did. However, happy to add it!

Reply:
Kind of a NAB to me and I'd almost learn towards not doing it. Maybe it's good to do just to get into the habit since we'd probably want to avoid this change if the app was widely adopted.